pkg/kubelet/cm/devicemanager/pod_devices.go

```diff
@@ -254,28 +255,28 @@ func (pdev *podDevices) deviceRunContainerOptions(podUID, contName string) *Devi
 		// Updates RunContainerOptions.Envs.
 		for k, v := range resp.Envs {
 			if e, ok := envsMap[k]; ok {
-				klog.V(4).Infof("Skip existing env %s %s", k, v)
+				klog.V(4).InfoS("Skip existing env", "envKey", k, "envValue", v)
 				if e != v {
-					klog.Errorf("Environment variable %s has conflicting setting: %s and %s", k, e, v)
+					klog.ErrorS(nil, "Environment variable has conflicting setting", "envKey", k, "expected", v, "got", e)
 				}
 				continue
 			}
-			klog.V(4).Infof("Add env %s %s", k, v)
+			klog.V(4).InfoS("Add env", "envKey", k, "envValue", v)
 			envsMap[k] = v
 			opts.Envs = append(opts.Envs, kubecontainer.EnvVar{Name: k, Value: v})
 		}
 
 		// Updates RunContainerOptions.Devices.
 		for _, dev := range resp.Devices {
 			if d, ok := devsMap[dev.ContainerPath]; ok {
-				klog.V(4).Infof("Skip existing device %s %s", dev.ContainerPath, dev.HostPath)
+				klog.V(4).InfoS("Skip existing device.", "containerPath", dev.ContainerPath, "hostPath", dev.HostPath)
 				if d != dev.HostPath {
-					klog.Errorf("Container device %s has conflicting mapping host devices: %s and %s",
-						dev.ContainerPath, d, dev.HostPath)
+					klog.ErrorS(nil, "Container device has conflicting mapping host devices.",
+						"path", dev.ContainerPath, "expected", d, "got", dev.HostPath)
```

Comment:
```suggestion
					klog.ErrorS(nil, "Container device has conflicting mapping host devices.",
						"containerPath", dev.ContainerPath, "expected", d, "got", dev.HostPath)
```

Reply:
https://github.com/kubernetes/kubernetes/pull/99976#discussion_r590295462 this conflicts with later suggestion. will keep as it is, since other places are following this pattern

Reply:
correct, had added the suggestion looking at line no 272 above, but I agree in this message there is only one key referring path and it doesnt need to be explicit (`containerPath`)

Reply:
I think this should remain `path`.